draft-ietf-mls-protocol.md

```diff
@@ -2815,7 +2860,9 @@ welcome_key = KDF.Expand(welcome_secret, "key", key_length)
     children are non-empty and have the hash of this node set as their
     `parent_hash` value (if the child is another parent) or has a `parent_hash`
     extension in the KeyPackage containing the same value (if the child is a
-    leaf).
+    leaf). If either of the node's children is empty, and in particular does not
```

Comment:
```suggestion
    leaf). If either of the node's children are empty, and in particular does not
```

Reply:
No, the subject which which the verb must agree is "either" (singular), not "children" (plural).